app/code/Magebit/Faq1/Model/QuestionManagement.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Model;
+
+use Magebit\Faq1\Api\QuestionManagementInterface;
+use Magebit\Faq1\Api\QuestionRepositoryInterface;
+use Magento\Framework\Exception\CouldNotSaveException;
+
+/**
+ * Management class that contains logic of disableQuestion and enableQuestion
+ */
+
+class QuestionManagement implements QuestionManagementInterface
+{
+
+    protected QuestionRepositoryInterface $questionRepository;
+
+    /**
+     * @param QuestionRepositoryInterface $questionRepository
+     */
+    public function __construct
+    (
+        QuestionRepositoryInterface $questionRepository
+    )
+    {
+        $this->questionRepository = $questionRepository;
+    }
+
+
+    /**
+     * @param $id
+     * @return bool
+     * @throws CouldNotSaveException
+     */
+    public function disableQuestion($id): bool
+    {
+        $question = $this->questionRepository->get($id);
+        $question->setStatus(0);
```

Comment:
move this "0" into QuestionInterace as a parameter, and then call it here, so that it isn't just an empty 0 here, but QuestionInterface::ENABLED